tests/driver_si70xx/main.c

```diff
@@ -44,25 +31,10 @@ int main(void)
 
     puts("SI7021 temperature and humidity sensor test application\n");
 
-    /* enable the sensor if test pin given */
-    if (TEST_PIN_EN != GPIO_UNDEF) {
```

Comment:
that's not the point, my understanding would be that the sensor has an _enable_ pin that requires a HIGH signal to set the device active. Depending on the wiring of the sensor module, it might be necessary to have this PIN connected and set HIGH - alternatively the sensor modules enable pin has to be connected to vcc.